marklogic-data-hub-central/ui/e2e/cypress/integration/curation/load/ingestionForAllFiletypes.spec.tsx

```diff
@@ -115,7 +115,7 @@ describe("Verify ingestion for all filetypes", () => {
     cy.waitForAsyncRequest();
     cy.verifyStepRunResult("success", "Ingestion", stepName);
     //Verify step name appears as a collection facet in explorer
-    runPage.explorerLink().click();
+    runPage.explorerLink().click({multiple: true, force: true});
```

Comment:
Do we need to allow multiple clicks? Did you try only using the force?

Reply:
You can also try running in open mode, and checking if maybe you need for something to finish loading, or scroll so that the element it's visible, just sharing some ideas

Reply:
Discussed with Camila that this was needed/okay for now, resolving.